e2e/scenario_test.go

```diff
@@ -919,3 +927,83 @@ func Scenario_ubuntu2204Wasm(t *testing.T) {
 		},
 	})
 }
+
+/*
+We could figure out the expected versions by parsing the components.json and manifest.json files, but then we'd need logic
```

Comment:
manifest.json doesn't exists anymore

Reply:
It sure does exist - and the code I found that installs containerd uses it. But there might be multiple places, so I'll have another hunt.

Reply:
Turns out I was looking at the self-contained VHD install code. That references manifest.json. However, the main script uses components.json. Manifest.json and manifest.cue still exist in the repository tho - they haven't been deleted.